piper/buttondialog.py

```diff
@@ -230,6 +258,16 @@ def _on_row_activated(self, listbox, row):
             self._action_type = row._action_type
             self._mapping = row._value
 
```

Comment:
I'm gonna have to be annoying here but: this is not good style. If you have a function that does one of two things based on a condition and **both** blocks are reasonably complicated (i.e. more than two lines), split it into helper functions and call them like this:
````
def init_ui():
   if button.index in [0, 1]:
     self._init_primary_buttons_ui()
   else:
     self._init_other_buttons_ui()
````
Much more readable than the semi-hidden `return` you have here.